docker-app/worker_wrapper/wrapper.py

```diff
@@ -45,8 +44,8 @@
 RETRY_COUNT = 5
 TIMEOUT_ERROR_EXIT_CODE = -1
 DOCKER_SIGKILL_EXIT_CODE = 137
-QFIELDCLOUD_QGIS_IMAGE_NAME = os.environ.get("QFIELDCLOUD_QGIS_IMAGE_NAME", None)
-QFIELDCLOUD_HOST = os.environ.get("QFIELDCLOUD_HOST", None)
+QFIELDCLOUD_QGIS_IMAGE_NAME = settings.QFIELDCLOUD_QGIS_IMAGE_NAME
+QFIELDCLOUD_HOST = settings.QFIELDCLOUD_HOST
```

Comment:
```suggestion
```

I think these can easily be retired as variables, all can be used is directly `settings.QFIELDCLOUD....`

Reply:
As we discussed, it is required down there, but can be replaced with `settings.QFIELDCLOUD...`. Same goes for assert 3 lines below.